data/human/outfits.txt

```diff
@@ -252,9 +252,9 @@ outfit "Surveillance Pod"
 	"mass" 5
 	"outfit space" -5
 	"outfit scan power" 16
-	"outfit scan speed" 2
+	"outfit scan speed" 50
 	"cargo scan power" 4
-	"cargo scan speed" 2
+	"cargo scan speed" 18
```

Comment:
There are other scanners in the game beyond just these human ones that would need adjusted.

Reply:
Given that increased power doesn't inherently reduce actual scan speed anymore, are these adjustments still necessary anyway?

Reply:
I suppose we could get away with leaving the scan speeds where they're at.

Right now this cargo scanner takes about 0.7 seconds to scan any ship. If its stats were left alone, then you're looking at 7 seconds at the quickest with against a ship with a cargo space of 200. But given that increased distance reduces scan speed, you might actually get more like 14 seconds against a 200 cargo space ship, and smaller ships would be even quicker. And even if 14 seconds seems too long to stay on target, at least the decay rate is much less proportional to the total scan time needed.